README.md

```diff
@@ -66,7 +66,7 @@ This MCP server provides the following tools:
 
 5. Run the server:
    ```bash
-   uv run server.py
+   uv run --env-file .env server.py
```

Comment:
Need to provide it with the `.env` file to actually load the environment variables.